content/posts/irods-development-update-november-2022.md

```diff
@@ -0,0 +1,24 @@
+Title: iRODS Development Update: November 2022
+Date: 2022-11-30 12:00
+Author: Kory Draughn
+Slug: irods-development-update-november-2022
+Status: published
+
+
+We went to Supercomputing 2022!
+
+We got a chance to talk with Consortium members in person and there was a lot of booth traffic. Overall, it was a success!
+
+Okay, enough about Supercomputing. Let's get into the update.
+
+For the server, we've been working hard to resolve several issues found in the 4.3.0 server. At the top of that list are memory leaks. The server has received some important changes in regard to that. You can follow that work at [https://github.com/irods/irods/pull/6673](https://github.com/irods/irods/pull/6673).
```

Comment:
"memory leaks" -> "memory leaks associated with the new cron manager"

"some important changes and you can follow"

Reply:
The memory leaks are specific to the cron manager. The are across various APIs.

Reply:
Updated. Let me know if anything needs to be changed. Will mark as resolved if good.

Reply:
resolving.